recipes/mosquitto/2.x/conanfile.py

```diff
@@ -139,9 +110,9 @@ def package(self):
             rm(self, "*.dylib", os.path.join(self.package_folder, "lib"))
             rm(self, "*.dll", os.path.join(self.package_folder, "bin"))
         elif self.options.shared and self.settings.compiler == "Visual Studio":
-            self.copy("mosquitto.lib", src=os.path.join(self._build_subfolder, "lib"), dst="lib")
+            copy(self, "mosquitto.lib", src=os.path.join(self.folders.build, "lib"), dst="lib")
             if self.options.build_cpp:
-                self.copy("mosquittopp.lib", src=os.path.join(self._build_subfolder, "lib"), dst="lib")
+                copy(self, "mosquittopp.lib", src=os.path.join(self.folders.build, "lib"), dst="lib")
 
     def package_info(self):
```

Comment:
There are some pkg_config properties that need to be updated for the new Generators :)